graphs/adapter-declaration/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/declaration/declaration/declareDirectedGraph.kt

```diff
@@ -1,6 +1,6 @@
 package com.anaplan.engineering.azuki.graphs.adapter.declaration.declaration
 
-data class DirectedGraphDeclaration<T>(
+data class declareDirectedGraph<T>(
```

Comment:
🚫 I don't get what your intention is here. This is a class which holds declaration data, not a function which actually performs the declaration. Please revert this change.

Reply:
https://github.com/anaplan-engineering/azuki/pull/40#discussion_r1576042326 I got the impression this what what you where looking for with this comment but I belive I was mistaken.